governance/parameter-docs/param-dai-savings-rate.md

```diff
@@ -0,0 +1,50 @@
+# Dai Savings Rate
+
+```
+Alias: DSR
+Parameter Name: Pot_dsr
+Containing Contract: Pot
+Scope: System
+Technical Docs: https://docs.makerdao.com/mcd-developer-guides/developer-guides-and-tutorials#dai-savings-rate-dsr
+```
+
+## Description
+The Dai Savings Rate is a tool that Maker Governance can use to adjust the interest rate paid to DAI holders who have deposited DAI into the Dai Savings Rate contract (`Pot`). Maker Governance directly controls the Dai Savings Rate.
+
+The Dai Savings Rate is usually expressed as a percentage and can be set to any value. However, even though Maker Governance can set the Dai Savings Rate to a negative value, this will not affect the balances held within the `Pot`. The balances within the `Pot` will not decrease because the drip function will revert if negative.
```

Comment:
Feels like this goes into a little too much detail for this section. The key information here is that:
* It's expressed as a percentage as an APR/APY (I forget which of these is correct)
* It's valid for any positive value.

Reply:
Maybe give an idiot-proof example here as well. If the DSR is set to 1%, and a DAI Holder has locked 100 DAI in the DSR for the entire year, they should expect to have 101 DAI after a year. (Check that these numbers are correct though, because they depend on whether it's compounding, which I _think_ it is.)

Reply:
Noted and updated. It's an APY and is a compounding rate per: https://docs.makerdao.com/smart-contract-modules/rates-module#dai-savings-rate-accumulation